migrations/founders.json

```diff
@@ -1,13 +1,33 @@
 {
   "founders": {
     "ganache": [
+      {
```

Comment:
why you need to mention the  'ganache' ?
founders are founders are founders .

Reply:
@dkent600 ?

Reply:
This file contains both ganache and kovan founder data.

Reply:
why ?